mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping

```diff
@@ -34,5 +34,5 @@ CLASS net/minecraft/class_2609 net/minecraft/block/entity/AbstractFurnaceBlockEn
 	METHOD method_17760 dropExperience (Lnet/minecraft/class_1937;Lnet/minecraft/class_243;IF)V
 	METHOD method_17763 dropExperience (Lnet/minecraft/class_1657;)V
 		ARG 1 player
-	METHOD method_26395 isFlammableWood (Lnet/minecraft/class_1792;)Z
+	METHOD method_26395 isNonFlammableWood (Lnet/minecraft/class_1792;)Z
```

Comment:
```suggestion
	METHOD method_26395 isFireproofWood (Lnet/minecraft/class_1792;)Z
```

Reply:
The tag it checks is `non_flammable_wood`.